python/dgl/subgraph.py

```diff
@@ -163,6 +163,8 @@ def _process_nodes(ntype, v):
     ]
     sgi = graph._graph.node_subgraph(induced_nodes, relabel_nodes)
     induced_edges = sgi.induced_edges
+    if not relabel_nodes:
+        sgi = graph._graph.edge_subgraph(induced_edges, (not relabel_nodes))
```

Comment:
`relabel_nodes=False`

Reply:
OK, fixed it. I just write true here because the name for the second paramenter is not relabel_nodes.